test/unit/parser-test.js

```diff
@@ -0,0 +1,122 @@
+define([
+    'utils/parser'
+], function (parser) {
+    /* jshint qunit: true */
+
+    module('parser');
+
+    var testerGenerator = function (assert, method) {
+        return function (left, right, message) {
+            assert.strictEqual(method.apply(this, left), right, message);
+        };
+    };
+
+    test('parser.getAbsolutePath', function(assert) {
+        var path = parser.getAbsolutePath(null, null);
+        assert.notOk(path);
```

Comment:
path is false or an empty string? Explain with a description:

`assert.notOk(path, 'passing null values returns X');`